src/DynamicModel.cpp

```diff
@@ -40,6 +40,31 @@ base::Vector6d DynamicModel::calcAcceleration(const base::Vector6d &control_inpu
     return invert_inertia_matrix*acceleration;
 }
 
+base::Vector6d DynamicModel::calcEfforts(const base::Vector6d& acceleration, const base::Vector6d& velocity, const base::Orientation& orientation) const
+{
+    // Check inputs
+    checkAcceleration(acceleration);
+    checkVelocity(velocity);
+
+    // Calculating the efforts given the current state based on all the hydrodynamics effects
+    base::Vector6d efforts = base::Vector6d::Zero();
+
+    efforts = uwv_parameters.inertia_matrix * acceleration + calcGravityBuoyancy(orientation, uwv_parameters);
+    switch(uwv_parameters.model_type)
```

Comment:
I mean this [part](https://github.com/rock-control/control-uwv_dynamic_model/blob/b2346d833a8ab3aca53c5a8a30bf997c6f43717e/src/DynamicModel.cpp#L30).

Looks like both methods (calcAcceleration and calcEfforts) need a common term (coriolis, damping and gravity effects).
 
What about having a `getTypeEffort()` method to deal with the model_type and the `switch`?